--- Original
+++ Patched
@@ -31,6 +31,7 @@
 
 PLAYERS_SRC = $(wildcard AI*.cc)
 PLAYERS_OBJ = $(patsubst %.cc, %.o, $(PLAYERS_SRC)) $(EXTRA_OBJS) $(DUMMY_OBJ)
+PLAYERS_SHARE_OBJ = $(patsubst %.o, %.share.o, $(PLAYERS_OBJ))
 
 # Flags
 
@@ -70,3 +71,9 @@
 	$(CXX) $(CXXFLAGS) -MM *.cc > Makefile.deps
 
 include Makefile.deps
+
+.PHONY: shareobjs
+shareobjs: $(PLAYERS_SHARE_OBJ)
+
+%.share.obj: %.o
+	strip $^ -o $@
